web_exportall/README.rst

```diff
@@ -0,0 +1,22 @@
+Description
```

Comment:
Please use the OCA's template for README: https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst